src/asim/configs/crossborder/constants.yaml

```diff
@@ -1,102 +0,0 @@
-## ActivitySim
-## See full license in LICENSE.txt.
-
-SanYsidro_maz_id: 9279
-OtayMesa_maz_id: 9387
-Tecate_maz_id: 22324
-OtayMesaEast_maz_id: 7123
-
-scenarioYear: 2022
-
-walkSpeed: 3.00
-
-# convenience for expression files
-HHT_NONFAMILY: [4, 5, 6, 7]
-HHT_FAMILY: [1, 2, 3]
-
-PSTUDENT_GRADE_OR_HIGH: 1
-PSTUDENT_UNIVERSITY: 2
-PSTUDENT_NOT: 3
-
-GRADE_SCHOOL_MAX_AGE: 14
-GRADE_SCHOOL_MIN_AGE: 5
-
-SCHOOL_SEGMENT_NONE: 0
-SCHOOL_SEGMENT_GRADE: 1
-SCHOOL_SEGMENT_HIGH: 2
-SCHOOL_SEGMENT_UNIV: 3
-
-PEMPLOY_FULL: 1
-PEMPLOY_PART: 2
-PEMPLOY_NOT: 3
-PEMPLOY_CHILD: 4
-
-PTYPE_FULL: &ptype_full 1
-PTYPE_PART: &ptype_part 2
-PTYPE_UNIVERSITY: &ptype_university 3
-PTYPE_NONWORK: &ptype_nonwork 4
-PTYPE_RETIRED: &ptype_retired 5
-PTYPE_DRIVING: &ptype_driving 6
-PTYPE_SCHOOL: &ptype_school 7
-PTYPE_PRESCHOOL:  &ptype_preschool 8
-
-CDAP_ACTIVITY_MANDATORY: M
-CDAP_ACTIVITY_NONMANDATORY: N
-CDAP_ACTIVITY_HOME: H
-
-# SHARED2 and SHARED3 Occupancies
-OCC_SHARED2: 2.0
-OCC_SHARED3: 3.33
-# add TNC and Taxi Occupancies
-OCC_TAXI: 1.1
-OCC_TNC_SINGLE: 1.2
-OCC_TNC_SHARED: 2.0
-
-# Trip Mode Choice
-cost_share_s2: 2
```

Comment:
cost_share_s2 and cost_share_s2 values differ between common and crossborder constants.yaml. Need to update crossborder trip_mode_choice_annotate_trips_preprocessor.csv and write_trip_matrices_annotate_trips_preprocessor.csv to divide by this value rather than multiply (consistent with resident configs)